Bricks/file/file.h

```diff
@@ -214,79 +214,125 @@ struct FileSystem {
   // TODO(dkorolev): Make OutputFile not as tightly coupled with std::ofstream as it is now.
   typedef std::ofstream OutputFile;
 
+  struct ScanDirItemInfo {
+    const std::string name;
+    const std::string path;
```

Comment:
I'd propose `basename`/`pathname` naming convention here.
https://en.wikipedia.org/wiki/Basename
@dkorolev what do you think?